src/main.js

```diff
@@ -69,7 +72,7 @@ const createWindow = async () => {
 
   // Create the browser window.
   const { width, height } = screen.getPrimaryDisplay().workAreaSize
-  console.log(width + ' ' + height)
+  logger.debug(width + ' ' + height)
```

Comment:
Is there a recommended way to do string replacement for log messages?  In other parts of this PR, for example, the `${variable}` notation is used instead.  Is one preferable for this use case?

I know we're only writing this to a logfile at the moment, but it could also behoove us to think forward to the possibility of sending log messages somewhere else.  In python's logging module, for example, the C-style '%s' notation was recommended to guard against code injection in the same way that MySQL queries support it as well.

Reply:
Well this particular message can be nixed. But to answer your question, winston has C-style interpolation that can be enabled with a format option https://github.com/winstonjs/winston#string-interpolation

But elsewhere in the docs they say template strings are the way to go. https://github.com/winstonjs/winston#formats

What do you think?

Reply:
For the sake of readability, I think the template strings are the way to go!

I think my concern with guarding against code injection is also a bit misguided as well and unnecessary for this use case since there's no opportunity for remote execution.  If someone injects code into the application, the worst that'll happen at this point is that they'll compromise their own computer.